python/mxnet/gluon/parameter.py

```diff
@@ -644,7 +644,7 @@ def var(self):
         """Returns a symbol representing this parameter."""
         if self._var is None:
             if self._var_name is None:  # _var_name is set manually in SymbolBlock.import
-                self._var_name = self._uuid
+                self._var_name = self._uuid.replace('-', '_') + '_' + self._name
```

Comment:
You need to document that `src/profiler/storage_profiler.cc` depends on this scheme.